internal/command/run/run.go

```diff
@@ -19,7 +21,35 @@ type Command struct {
 	Port string
 }
 
+func frontEnd() string {
+	// TODO: Change to the corresponding front end in next releases
+	return "Svelte " + version.Svelte
+}
+
+func displayDashboard(host, port string, timeElapsed time.Duration) {
+	/*
+		The dashboard should looks something like this:
+			|   bud dev server is running:
```

Comment:
I'd prefer to exclude this information since I think you'll learn that you're running the dev server fairly quickly and never forget it.